Example/GiniVision/ComponentAPIAnalysisViewController.swift

```diff
@@ -10,6 +10,13 @@ import UIKit
 import GiniVision
 import Gini_iOS_SDK
 
+protocol ComponentAPIAnalysisScreenDelegate:class {
+    func didCancelAnalysis()
```

Comment:
Just a question - why don't you use the conventional Cocoa framework delegate style? Here, it would have been:

`analysisScreenDidCancelAnalysis(_ screen: ComponentAPIAnalysisViewController)`

Reply:
I don't know, I see it simpler but yes, I should use Cocoa framework delegate style.